src/Autofac/Builder/RegistrationBuilder{TLimit,TActivatorData,TRegistrationStyle}.cs

```diff
@@ -366,13 +366,28 @@ public RegistrationBuilder(Service defaultService, TActivatorData activatorData,
         /// <returns>A registration builder allowing further configuration of the component.</returns>
         public IRegistrationBuilder<TLimit, TActivatorData, TRegistrationStyle> PropertiesAutowired(PropertyWiringOptions wiringFlags)
```

Comment:
I'd be OK with that. It is a 4.0 release and that seems like a legit breaking change. Not sure it'd actually break anyone since the number of folks implementing their own registration builder should be pretty low.